### PR TITLE
Flexbox

### PR DESCRIPTION
**Pull request by svyatoslavPy**
- [DEMO LINK](https://svyatoslavPy.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://svyatoslavPy.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, there are no demo link in the PR, also you need to add these links to the PR description
image

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link nav__link--hover"
+              href="#laptops&computers"  data-qa="hover">
+              Laptops & Computers
+            </a>
```
Comment (IvanFesenko) [inline src/index.html:40]: No need in `--hover` modifier. If tag have mpre than 2 attributes - it each attribute should be on new line, closed bracket should be on the same line as opebn
```suggestion
            <a class="nav__link"
              href="#laptops&computers"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,93 @@
...
+  font-weight: 500;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  transition: 0.1s;
+}
+
+.nav__link:hover {
+  cursor: pointer;
+  color: var(--primary-color);
+}
```
Comment (IvanFesenko) [inline src/style.css:67]:
```suggestion
  padding-bottom: 23px;
  transition: 0.1s;
}

.nav__link:hover {
  color: var(--primary-color);
}
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,93 @@
...
+
+.nav__link--hover {
+  color: var(--primary-color);
+}
```
Comment (IvanFesenko) [inline src/style.css:71]:
```suggestion

```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,93 @@
...
+
+@media screen and (max-width: 1200px) {
+  .nav__link--hover {
+    color: var(--secondary-color);
+  }
+}
```
Comment (IvanFesenko) [inline src/style.css:77]: Redundant
```suggestion

```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,93 @@
...
+
+.nav__link {
+  display: inline-block;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: var(--secondary-color);
+  font-size: 12px;
+  font-weight: 500;
+  padding-top: 23px;
+  padding-bottom: 23px;
```
Comment (IvanFesenko) [inline src/style.css:60]: Use short hand
```suggestion
  padding: 23px 0;
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Looks good, Check my comments

File: src/index.html [hunk 6]
```diff
@@ -1,14 +1,73 @@
...
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link is-hover"
```
Comment (IvanFesenko) [inline src/index.html:41]: is-hover - is redundant
```suggestion
            <a class="nav__link"
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,92 @@
...
+.is-hover {
+  color: var(--primary-color);
+}
+
+
+@media screen and (max-width: 1200px) {
+  .is-hover {
+    color: var(--secondary-color);
+  }
+}
```
Comment (IvanFesenko) [inline src/style.css:76]:
```suggestion

```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,92 @@
...
+  text-decoration: none;
+  color: var(--secondary-color);
+  font-size: 12px;
+  font-weight: 500;
+  padding: 23px 0;
+  transition: 0.1s;
+}
+
+.nav__link:hover {
+  cursor: pointer;
```
Comment (IvanFesenko) [inline src/style.css:63]: tag `a` by defaul has cursor: pointer - redundant
```suggestion

```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Remove redundant class - if you have questions - ping me in fe_chat

**Review by IvanFesenko — APPROVED** [review]
Well done

**Pull request closed without merging**